x/programs/cmd/simulator/src/lib.rs

```diff
@@ -326,10 +359,6 @@ mod tests {
         let output_json = serde_json::to_value(&param).unwrap();
 
         assert_eq!(output_json, expected_json);
-
-        let output_param: Param = serde_json::from_value(expected_json).unwrap();
-
-        assert_eq!(output_param, expected_param);
```

Comment:
We never needed to deserialize a `Param`, so I removed the derive attribute as well as the test